ui/admin/app/routes/onboarding/success.js

```diff
@@ -0,0 +1,29 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+export default class OnboardingSuccessRoute extends Route {
+  // =services
+  @service router;
+  @service session;
+  @service browserObject;
+
+
+  // =methods
+  model() {
+    const model = this.modelFor('onboarding');
+    return {
+      ...model,
+      clusterUrl: this.browserObject.window.origin,
+    };
+  }
+
+  @action
+  showTargetList(scope) {
+    const {
+      project: { id: scopeID },
+      target: { id: targetID },
+    } = scope;
+    this.router.transitionTo('scopes.scope.targets.target', scopeID, targetID);
```

Comment:
What does the `targetId` do here? I understand we pass scopeID as the model; do we need to pass in targetID as well?

Reply:
Yes because both scope and target have dynamic parts.